.gitignore

```diff
@@ -34,6 +34,12 @@
 # CMake build directory
 build/
 cmake-build-*/
+CMakeFiles/
+cmake_install.cmake
+CMakeCache.txt
+Makefile
+demo
+run_tests
```

Comment:
This seems a bit drastic. I would recommend that you move your build files into a separate directory, e.g. `build`, which is already gitignored.

Reply:
All these files are created by running the travis commands for GCC 7 locally for Libcanard.
```bash
CC=gcc-7 && CXX=g++-7 && cd tests/ && cmake . && make
./run_tests --rng-seed time
```

In my opinion, these files should therefore be included in the gitignore as they will be present for all peoples which want to run tests locally before committing code.

Reply:
I didn't expect anyone to run them locally. The Travis script should be updated then.